src/uproot/exceptions.py

```diff
@@ -63,25 +62,25 @@ def __str__(self):
                     break
             if to_show is None:
                 to_show = "(none!)"
-            with_keys = "\n\n    Available keys: {0}\n".format(to_show)
+            with_keys = f"\n\n    Available keys: {to_show}\n"
 
         in_file = ""
         if self.file_path is not None:
-            in_file = "\nin file {0}".format(self.file_path)
+            in_file = f"\nin file {self.file_path}"
 
         in_object = ""
         if self.object_path is not None:
-            in_object = "\nin object {0}".format(self.object_path)
+            in_object = f"\nin object {self.object_path}"
 
         if self.cycle == "any":
-            return """not found: {0} (with any cycle number){1}{2}{3}{4}""".format(
+            return """not found: {} (with any cycle number){}{}{}{}""".format(
                 repr(self.key), because, with_keys, in_file, in_object
             )
         elif self.cycle is None:
-            return """not found: {0}{1}{2}{3}{4}""".format(
+            return """not found: {}{}{}{}{}""".format(
```

Comment:
Something like this would be nicer nicer as an f-string too, but pyupgrade is too frugal.

Reply:
If the expressions on the next line are long, it may be best as it is. As a `.format`, they don't have to maintain zero whitespace between expressions.

Reply:
Yes, but if it fits in one line, then it's best in the original line (especially when they are all together like this). If there are more than a line, or if they have expressions other than just simple values, yes. (I haven't tried putting these in to see if it would overflow a line)